File: 백준/Bronze/9498. 시험 성적/시험 성적.swift
import Foundation

let score = Int(readLine()!)!

switch score {
case 90...:
    print("A")
case 80 ... 89:
    print("B")
case 70 ... 79:
    print("C")
case 60 ... 69:
    print("D")
default:
    print("F")
}

File: 백준/Bronze/9498. 시험 성적/시험 성적.py
import sys
readLine = sys.stdin.readline
score = int(readLine())
match score:
    case _ if score >= 90:
        print("A")
    case _ if score >= 80:
        print("B")
    case _ if score >= 70:
        print("C")
    case _ if score >= 60:
        print("D")
    case _:
        print("F")
